plonecli/cli.py

```diff
@@ -216,16 +216,44 @@ def run_serve(context):
 
 @cli.command('test')
 @click.option('-a', '--all', 'all', is_flag=True)
+@click.option('-t', '--test', 'test', type=click.Path())
+@click.option('-s', '--package', 'package')
 @click.pass_context
-def run_test(context, all):
+def run_test(context, all, test, package):
     """Run the tests in your package"""
     if context.obj.get('target_dir', None) is None:
         raise NotInPackageError(context.command.name)
     params = [
         './bin/test',
     ]
-    if all:
+    if test and package and all:
```

Comment:
This looks like, we could just add every param if it exist, no?

Reply:
Not sure, there was some fault with this approach. I couldn't remember that now. But it's working...😅